src/clients/services/gatewayClient.ts

```diff
@@ -13,6 +13,7 @@ export class GatewayClient extends HttpClient {
 
   public async upload(buffer: Buffer): Promise<void> {
     const formData = new FormData();
+    // TODO: fix in integration
```

Comment:
you didn't resolved my remark of "i dont think you need to specify the baseUrl"

Reply:
fixed